ray-operator/config/samples/ray-job.interactive-mode.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: ray.io/v1
+kind: RayJob
+metadata:
+  name: rayjob-interactive-mode
+  namespace: default
+spec:
+  # InteractiveMode is only used in kubectl plugin
```

Comment:
Personally I think we can document `submissionMode: InteractiveMode` as something you can use even outside kubectl plugin. And if this is the case, the example should include setting a pre-defined `spec.submissionID`

Reply:
done